docs/cli/nrfu.md

```diff
@@ -90,22 +96,21 @@ The `text` subcommand provides a straightforward text report for each test execu
 anta nrfu text --help
 Usage: anta nrfu text [OPTIONS]
 
-  ANTA command to check network states with text result
+  ANTA command to check network states with text result.
 
 Options:
-  -s, --search TEXT  Regular expression to search in both name and test
-  --skip-error       Hide tests in errors due to connectivity issue
-  --help             Show this message and exit.
+  --skip-error    Hide tests in errors due to connectivity issue
+  --skip-failure  Hide tests in failure
+  --skip-success  Hide tests in success to focus on error or failure
+  --help          Show this message and exit.
 ```
 
-The `--search` option permits filtering based on a regular expression pattern in both the hostname and the test name.
-
-The `--skip-error` option can be used to exclude tests that failed due to connectivity issues or unsupported commands.
+The `--skip-error` option can be used to exclude tests that failed due to connectivity issues or unsupported commands. Same behavior applied to `--skip-failure` to skip tests in failure and `--skip-success` to skip tests in success.
 
 ### Example
 
 ```bash
-anta nrfu text --tags LEAF --search DC1-LEAF1A
+anta nrfu --device DC1-LEAF1A text
```

Comment:
do we still have an example with tags somewhere?

Reply:
In all other outputs